.gitignore

```diff
@@ -1,4 +0,0 @@
-_site/
-.sass-cache/
-.jekyll-metadata
-*~
```

Comment:
I totally understand what's happened so far and it's totally understandable. You're really close. I don't think it will be easy for me to explain via this comment, but I will try. 

There is a short section on branching in the Effective Computation in Physics book, but it doesn't address the particular predicament you're in. Namely, you created a branch for radioactive-decay based on the master branch. Then, you created the atomic-structure branch off of the radioactive-decay branch (this is the initial problem) and manually, with ```git rm``` made it look like it was actually branched off of the master branch (this complicates the problem). In the future, a general rule is that it's cleanest to create branches that branch from master, not from other experimental branches. 

A complicated thing that probably helped lead to all of this: your master branch appears to be 10 commits ahead of necx-org/master. A best practice is to keep your master branch "clean". That is, it should only contain commits that have been merged into upstream (it should be kept identical to necx-org/master). I can walk through a bunch of fixes with you over google hangouts at some point, where you can share your screen and we can clean everything up together... let me know if that would be helpful.